README.md

```diff
@@ -108,14 +108,14 @@ All function calls (create, resolve, update, deactivate) can be called with an o
 }
 ```
 
-**chain** - the chain id or the registered chain name (see the [DID method schema](https://github.com/Gimly-Blockchain/eosio-did-spec#3-did-method-schema-dideosio) part of the EOSIO DID spec). This must be provided to know which chain to contact and a corresponding item in the [eosio-did-chain-registry.json](https://github.com/Gimly-Blockchain/eosio-did-resolver/blob/master/src/eosio-did-chain-registry.json) or `registry` property must exist e.g.
+**chain** - the chain id or the registered chain name (see the [DID method schema](https://github.com/Gimly-Blockchain/antelope-did-spec#3-did-method-schema-didantelope) part of the Antelope DID spec). This must be provided to know which chain to contact and a corresponding item in the [antelope-did-chain-registry.json](https://github.com/Gimly-Blockchain/antelope-did-resolver/blob/master/src/antelope-did-chain-registry.json) or `registry` property must exist e.g.
```

Comment:
also change this URL, change **all URLs pointing to the repositories that we are forking**